infra/discovery/src/listener/listener.js

```diff
@@ -236,7 +236,7 @@ logger.info(
   )}`
 )
 
-setNetwork(config.network)
+setNetwork(config.network, { performanceMode: false })
```

Comment:
Good catch! So that was causing the listener to talk to graphql.originprotocol.com in prod ?

Reply:
I think so.  It isn't guaranteed to cause problems, but I thought it best to make this change to prevent any weirdness, or to run into those same caching issues we had before.